spec/observables/dom/ajax-spec.ts

```diff
@@ -1140,19 +1133,23 @@ class MockXMLHttpRequest {
   }
 
   static clearRequest(): void {
+    MockXMLHttpRequest.noResponseProp = false;
     MockXMLHttpRequest.requests.length = 0;
     MockXMLHttpRequest.recentRequest = null!;
   }
 
   protected responseType: string = '';
   private readyState: number = 0;
 
-  private async: boolean = true;
+  /**
+   * Used to test if `open` was called with `async` true or false.
+   */
+  public async: boolean = true;
```

Comment:
Made this public so we can test it. (self nit: I should have removed the `public` and just let it be public, but we should probably use a lint rule for that)